Gruntfile.js

```diff
@@ -46,59 +46,26 @@ module.exports = function(grunt) {
       }
     },
 
-    styleguide: {
-            dist: {
-                options: {
-                    name: 'eRegs Styleguide'
-                },
-                files: {
-                    '<%= env.frontEndPath %>/docs/styleguide': '<%= env.frontEndPath %>/css/'
-                }
-            }
-        },
-
     /**
-     * JSHint: https://github.com/gruntjs/grunt-contrib-jshint
+     * ESLint: https://github.com/sindresorhus/grunt-eslint
```

Comment:
Hm, this might be a case where the `grunt-contrib-` version is way less used... cc @Scotchester 

`19,290 downloads in the last month` https://www.npmjs.com/package/grunt-eslint
`435 downloads in the last month` https://www.npmjs.com/package/grunt-contrib-eslint